lib/v2/datasource/remote/model/profile_model.dart

```diff
@@ -25,6 +25,12 @@ class ProfileModel {
     this.timestamp,
   });
 
+  /// Returns the account age in days
+  int get accountAge {
+    var creationDate = DateTime.fromMillisecondsSinceEpoch(timestamp! * 1000);
+    return DateTime.now().difference(creationDate).inDays;
+  }
+
   ProfileModel copyWith({
     String? account,
```

Comment:
i think we can make account not nullable, since to be able to load this page we must have an account.

Reply:
Only the account field?